includes/class-cache.php

```
@@ -627,8 +651,51 @@ private function core_will_inline( $handle ): bool {
+				$this->log_inline_budget_drift( $handle, $limit );
+
+				// Conservative downgrade: assume core WILL inline the style. Leaving it
+				// out of the combined file is safe in every direction — either core
+				// inlines it (correct), or it is served as its own external <link>
+				// (a minor perf loss, never duplicated rules). Returning false here
+				// would risk pulling an inlined style into the combined file.
+				return true;
+			}
+
+			return $prediction;
+		}
```

PHPMD (2.15.0)

[error] The method core_will_inline is not named in camelCase. (undefined)

(CamelCaseMethodName)